### PR TITLE
PHPStan passing at level 1

### PR DESCRIPTION
**Pull request by dstockto**
These changes get us to passing PHPStan level 1 - mostly. There's a couple files in scripts, and one function defined in public/index.php that PHPStan calls out for autoloading.

### CODE REVIEW

**Comment by localheinz** [comment]
@dstockto 

What do you think about splitting this pull request into two commits:

- one that adds/adjusts the primary configuration
- one or more that fix the issues

?

File: phpstan.neon [hunk 1]
```diff
@@ -0,0 +1,8 @@
+parameters:
+    excludes_analyse:
+        - */public/index.php
+        - */scripts/*
+    paths:
```
Comment (localheinz) [inline phpstan.neon]: What do you think about keeping parameters sorted by name here?

This way it will be more obvious where to add new parameters.

File: src/Controller/EventsController.php [hunk 2]
```diff
@@ -14,10 +14,10 @@
 use Joindin\Api\Model\TalkMapper;
 use Joindin\Api\Model\TrackMapper;
 use Joindin\Api\Model\UserMapper;
+use Joindin\Api\Request;
 use Joindin\Api\Service\EventApprovedEmailService;
 use Joindin\Api\Service\EventSubmissionEmailService;
 use PDO;
-use Joindin\Api\Request;
```
Comment (localheinz) [inline src/Controller/EventsController.php]: Is this change related to increasing the `phpstan` level?

Comment (dstockto) [reply src/Controller/EventsController.php]: It is - one of the things it calls out is unused imports.

File: phpstan.neon (hunk 1 above)
Comment (dstockto) [reply phpstan.neon]: Sure, I can do that.

**Comment by dstockto** [comment]
> @dstockto
> 
> What do you think about splitting this pull request into two commits:
> 
> * one that adds/adjusts the primary configuration
> * one or more that fix the issues
> 
> ?

I can split this up into two.

**Comment by dstockto** [comment]
Configuration PR is here - https://github.com/joindin/joindin-api/pull/735

**Comment by dstockto** [comment]
#736 contains the code changes from this PR. Closing this one.

File: src/Controller/EventsController.php (hunk 2 above)
Comment (dstockto) [reply src/Controller/EventsController.php]: Hmm. Now I'm not so sure. I was running PHPStan, Phan and Psalm. I'm going to add it back.
